### DISCUSSION TITLE
1.23.0 + Wear OS 2.14.0

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#975] [General]
- **New feature**: Copy count is viewable when sorting by times copied
- **New feature**: System theme (black) option
- **General**: Title bar is now fixed when transparent status bar is disabled
- **General**: 58 new icons and improvements to existing icons
- **General**: Translation updates

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.23.0'>1.23.0 + Wear OS 2.14.0</a>.</em>

**Comment by yizigezi** [comment]
Does this version support Wear OS 2? My device is a Mi Watch and it doesn't seem to work
